api/pom.xml

```diff
@@ -190,10 +189,30 @@
         <plugins>
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <!-- This is the common/default-compile execution configuration -->
                 <configuration>
-                    <source>${maven.compiler.source}</source>
-                    <target>${maven.compiler.target}</target>
+                    <release>11</release>
+                    <compilerArgs>
+                        <arg>-Xlint:all</arg>
+                    </compilerArgs>
+                    <showDeprecation>true</showDeprecation>
+                    <showWarnings>true</showWarnings>
```

Comment:
The PR #141 has a review comment from Emily about it needing to be an MR JAR.  I don't really see why that is needed.  If that is required then I would expect something like this to be configured for the release 11 execution:

```
<outputDirectory>${project.build.outputDirectory}/META-INF/versions/9</outputDirectory>
``` 

If no MR JAR is required then we might as well use PR #141

Reply:
I agree; I don't see why a MR JAR would be needed here.  The guidelines referenced in the description indicate the following:
```
If your spec is not updating the API and producing a service release that needs to support Java SE 8, run two passes of the compiler-plugin to produce the SE 8 based content, followed by a pass to produce the SE 9 based module-info.class.
```
PR #141 appears to follow the guidelines exactly, producing Java 8 content, except for a Java 8 module-info.class.

The guidelines then provide an example of another project which seems to produce a MR JAR, but I don't understand what advantage that would have for enterprise beans, and would just double the size of the API JAR.

Reply:
#141 is also an MR jar, but it does not depend on the correctly modularizied transactions artifact which is seen when compiling that module-info:

[WARNING] /Users/starksm/Dev/JBoss/Jakarta/rh-ejb/api/src/main/java/module-info.java:[22,32] requires transitive directive for an automatic module

That is also why the jakarta.transactions transitive dependency on CDI is not seen and thus required to compile.